# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/public/ProtoKey.h
# Commit: 8e609f97c413
# Full Hash: 8e609f97c413a12f7042dfe80647bdd34f0400d8
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2019-11-01 16:13:11
# Regressor Bug: 1587096
# File Overlap Count: 1
# Description:
#   Bug 1587096 - Part 1: Add JS FinalizationGroupObject and related data structures r=sfink
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49943
# ==============================================================================

diff -r f273ec2ec0ae -r 8e609f97c413 js/public/ProtoKey.h
--- a/js/public/ProtoKey.h	Fri Nov 01 10:37:09 2019 +0000
+++ b/js/public/ProtoKey.h	Fri Nov 01 10:37:25 2019 +0000
@@ -140,7 +140,8 @@
   IMAGINARY(WasmInstance, dummy, dummy)                                      \
   IMAGINARY(WasmMemory, dummy, dummy)                                        \
   IMAGINARY(WasmTable, dummy, dummy)                                         \
-  IMAGINARY(WasmGlobal, dummy, dummy)
+  IMAGINARY(WasmGlobal, dummy, dummy)                                        \
+  REAL(FinalizationGroup, InitViaClassSpec, OCLASP(FinalizationGroup))
 
 #define JS_FOR_PROTOTYPES(REAL, IMAGINARY)                      \
   JS_FOR_PROTOTYPES_(REAL, IMAGINARY, IF_INTL(REAL, IMAGINARY), \